todo_app/data/trello_items.py

```diff
@@ -0,0 +1,46 @@
+import requests
+import os
+
+API_KEY = os.getenv('TRELLO_API_KEY')
+TOKEN = os.getenv('TRELLO_TOKEN')
+BOARD_ID = os.getenv('TRELLO_BOARD_ID')
+BASE_URL = "https://api.trello.com/1/"
+
+def convert_trello_list(trello_list):
+    items = []
+    for card in trello_list['cards']:
+        items.append(Item.from_trello_card(card, trello_list))
+    return items
+
+
+def get_to_do_items():
+    to_do_list = requests.get(f"{BASE_URL}boards/{BOARD_ID}/lists?key={API_KEY}&token={TOKEN}&cards=open").json()[0]
+    global to_do_list_id
+    to_do_list_id = to_do_list['id']
+    return convert_trello_list(to_do_list)
+
+
+def get_done_items():
+    done_list = requests.get(f"{BASE_URL}boards/{BOARD_ID}/lists?key={API_KEY}&token={TOKEN}&cards=open").json()[2]
+    global done_list_id
+    done_list_id = done_list['id']
```

Comment:
A global variable like this is risky. 
E.g. if the server restarts while a user has the page open (or if you host the app on multiple servers), then done_list_id won't be defined yet when they try to mark something complete.

If you are going to have global variables like this, it's better to have an explicit initialise function that runs on startup.

If you are going to have state it might be clearer to wrap these functions in a class.